### PR TITLE
fix net.sf.json.JSONNull.contains()

### PR DESCRIPTION
**Pull request by v1v**
## What does this PR do?

fix generateBuildReport: net.sf.json.JSONNull

## Why is it important?

another corner case

## Related issues
Similar to https://github.com/elastic/apm-pipeline-library/pull/1617

### CODE REVIEW

File: src/test/resources/steps-errors-allowed-to-run.json [hunk 1]
```diff
@@ -1,4 +1,16 @@
 [
+  {
+    "displayDescription": null,
+    "displayName": "Force a null displayDesription",
```
Comment (mdelapenya) [inline src/test/resources/steps-errors-allowed-to-run.json:4]:
```suggestion
    "displayName": "Force a null displayDescription",
```

**Review by mdelapenya — APPROVED** [review]

**Comment by v1v** [comment]
:/ I wrongly tested my UT:

```
./mvnw test -Dtest=NotificationManagerStepTests#test_notify_pr_with_aborted_not_allowed_to_run -Pdebug
```

but missed to test all , so need to fix the breaking changes

File: src/test/resources/steps-errors-allowed-to-run.json [hunk 2]
```diff
@@ -1,4 +1,16 @@
...
+    "displayDescription": null,
+    "displayName": "Force a null displayDescription",
+    "durationInMillis": 23,
+    "id": "81",
+    "input": null,
+    "result": "FAILURE",
+    "startTime": "2021-09-22T09:32:09.201+0000",
+    "state": "FINISHED",
+    "type": "STEP",
+    "url": "https://apm-ci.elastic.co//blue/rest/organizations/jenkins/pipelines/apm-agent-nodejs/pipelines/ecs-logging-nodejs-mbp/pipelines/PR-101/runs/1/steps/81/log"
```
Comment (cachedout) [inline src/test/resources/steps-errors-allowed-to-run.json:12]:
```suggestion
    "url": "https://apm-ci.elastic.co/blue/rest/organizations/jenkins/pipelines/apm-agent-nodejs/pipelines/ecs-logging-nodejs-mbp/pipelines/PR-101/runs/1/steps/81/log"
```

Nit: double-slashes?

**Review by cachedout — APPROVED** [review]

**Review by kuisathaverat — APPROVED** [review]

**Comment by v1v** [comment]
Superseded by https://github.com/elastic/apm-pipeline-library/pull/1745
